htdocs/commande/class/commande.class.php

```diff
@@ -1325,6 +1325,9 @@ public function createFromProposal($object, User $user)
 			$line->marge_tx			= $marginInfos[1];
 			$line->marque_tx		= $marginInfos[2];
 
+			$line->origin           = $object->element;
```

Comment:
One question tho: as per the documentation I can read about `MAIN_CREATEFROM_KEEP_LINE_ORIGIN_INFORMATION` it looks like the element I should be passing here should be `$object->lines[$i]->element` and no the parent element. 

However in many triggers I'm using, when handling line it's always the parent element (propal, ...) instead of the line element (propaldet, ...) who I am receiving.